src/forwarders.js

```diff
@@ -1,6 +1,7 @@
 import Types from './types';
 import filteredMparticleUser from './filteredMparticleUser';
 import { isEmpty } from './utils';
+import FilterUtilities from './kitFilterHelper';
```

Comment:
We should rename `FilterUtilities` in all places to `KitFilterHelper` to match the class name and avoid confusion.

Reply:
Good catch.